app/controllers/boards_controller.rb

```diff
@@ -0,0 +1,51 @@
+class BoardsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :current_board, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @boards = Board.all
+  end
+
+  def show
+
+  end
+
+  def new
+    @board = Board.new
+  end
+
+  def create
+    board = Board.create(user_id: current_user.id, 
+      name: board_params['name'], 
+      description: board_params['description'])
```

Comment:
```suggestion
    board = Board.create!(user_id: current_user.id, 
                                         name: board_params['name'], 
                                         description: board_params['description'])
```

Reply:
also you can use the same board_params as in update action ( with the help of `merge` method )